sdk/formrecognizer/ai-form-recognizer/src/options/AnalyzeDocumentOptions.ts

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { OperationOptions } from "@azure/core-client";
+import {
+  AnalyzeResult,
+  AnalyzedDocument,
+  DocumentAnalysisPollOperationState,
+} from "../lro/analysis";
+import { PollerOptions } from "./PollerOptions";
+
+/**
+ * Add-on capabilities (features) that can be enabled for the request.
+ *
+ * For more information about the features available in Form Recognizer, see the service documentation:
+ *
+ * https://aka.ms/azsdk/formrecognizer/features
+ */
+export type FormRecognizerFeature =
+  | (typeof FormRecognizerFeature)[keyof typeof FormRecognizerFeature]
+  // eslint-disable-next-line @typescript-eslint/ban-types
+  | (string & {});
```

Comment:
:sunglasses: 

This is just used as input right?

Reply:
Yep! Input only.